protocols/mdns/src/behaviour/iface.rs

```diff
@@ -319,10 +316,9 @@ where
                 Poll::Ready(Ok(Err(err))) => {
                     tracing::debug!("Parsing mdns packet failed: {:?}", err);
                 }
-                Poll::Ready(Ok(Ok(None))) | Poll::Pending => {}
+                Poll::Ready(Ok(Ok(None))) => {}
```

Comment:
I'd prefer a `continue` here and restoring the others instead of explicitly returning from the `Poll::Pending` branch.

Reply:
Should be resolved now.